Code/GraphMol/SmilesParse/smarts.yy

```diff
@@ -484,6 +507,24 @@ atom_query:	simple_atom
 }
 ;
 
+possible_range_query : COMPLEX_ATOM_QUERY_TOKEN
+| HETERONEIGHBOR_ATOM_QUERY_TOKEN {
+  $1->setQuery(makeAtomNumHeteroatomNbrsQuery(0));
+}
+| ALIPHATICHETERONEIGHBOR_ATOM_QUERY_TOKEN {
+  $1->setQuery(makeAtomNumAliphaticHeteroatomNbrsQuery(0));
+}
+| RINGSIZE_ATOM_QUERY_TOKEN {
+  $1->setQuery(makeAtomMinRingSizeQuery(5)); // this is going to be ignored anyway
```

Comment:
Why is this being ignored?

Reply:
Why is the "default" 5?

Reply:
It’s ignored because the query “r” isn’t accepted on its own.
The 5 is just a number. It needs to be >=3 because `makeAtomMinRingSizeQuery()` doesn’t accept anything lower than 3.

Reply:
Might change the comment to be // the 5 in the ring size is required and will be ignored

But I'll go ahead and merge this.